packages/components/organisms/f-checkout/src/components/TermsAndConditions.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <div
+        data-test-id='terms-and-coniditions-component'
+        :class="$style['c-termsAndConditions']">
+        {{ $t('termsAndConditions.text1') }}
+        <a :href="$t('termsAndConditions.termsAndConditionsLinkUrl')">
+            <span>{{ $t('termsAndConditions.termsAndConditionsLinkText') }}</span>
+        </a>{{ $t('termsAndConditions.text2') }}
+        <a :href="$t('termsAndConditions.privacyPolicyLinkUrl')">
+            {{ $t('termsAndConditions.privacyPolicyLinkText') }}
+        </a>
+        {{ $t('termsAndConditions.text3') }}
+        <a :href="$t('termsAndConditions.cookiePolicyLinkUrl')">
+            <span>{{ $t('termsAndConditions.cookiePolicyLinkText') }}</span>
+        </a>.
+    </div>
```

Comment:
I believe this could be simplified significantly by using the `<i18n>` directive. Ping me if you need help with that.